directory.json

```diff
@@ -13,6 +13,7 @@
 	"BP": "https://raw.githubusercontent.com/fossasia/fossasia-communities/master/India/BangPypers-api.json",
 	"CALLUG":"https://raw.githubusercontent.com/fossasia/fossasia-communities/master/India/LinuxUsersGroup-Calcutta-api.json",
 	"CDLug": "https://raw.githubusercontent.com/fossasia/fossasia-communities/master/China/ChengduGNU-LinuxUserGroupCDLug-api.json",
+	"Ceylon": "https://raw.githubusercontent.com/fossasia/fossasia-communities/master/Sri%20Lanka/Ceylon-api.json",
```

Comment:
@roonyh : `"More importantly Ceylon Linux is not an open source organization. They are a private business which sells products for open source systems. I know this because I am from Sri Lanka and familiar with it :)"`